0-raw_data_processing/0-240409-E-MTAB-12916/0-240409-kallisto_bustools/0-240409-mapping_kallisto_bustools_E-MTAB-12916.ipynb

```diff
@@ -0,0 +1,1989 @@
+{
```

Comment:
<p>We might need to spend some time normalising the metadata information, to make it more compact for the final h5ad object. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/human_heart_mapping/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Compressed the metadata and added it to the anndata object in this file 0-raw_data_processing/0-240409-E-MTAB-12916/0-240409-kallisto_bustools/1-240415-merging_anndata_E-MTAB-12916.ipynb</p><p> </p>